buildlib/tools/perf_results.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+
+import sys
+
+def check_perf_results(filename, threshold):
```

Comment:
Shouldn't that be a PerfX functionality? We can set a threshold in CLI or YAML config and depends on that it will return a non-zero code for a failure.

Reply:
Absolutely! 
An external analyzer was a faster solution. If other projects may benefit from such functionality, let's make it a part of PerfX.

Reply:
Should we do it right now?

Reply:
Up to you. I need to align my priorities with the mgmt first.

Reply:
OK, let's see how it goes. Ideally we should not produce such scripts but put such functionality into the main tool. But it that is OK with UCX team, let's use it as a "temp solution" :)